core/modules/views/plugins/views_plugin_style_grid.inc

```diff
@@ -38,34 +38,58 @@ class views_plugin_style_grid extends views_plugin_style {
       '#required' => TRUE,
       '#min' => 1,
       '#step' => 1,
+      '#weight' => -2,
     );
     $form['alignment'] = array(
       '#type' => 'radios',
       '#title' => t('Alignment'),
       '#options' => array('horizontal' => t('Horizontal'), 'vertical' => t('Vertical')),
       '#default_value' => $this->options['alignment'],
-      '#description' => t('Horizontal alignment will place items starting in the upper left and moving right. Vertical alignment will place items starting in the upper left and moving down.'),
+    );
+    $form['alignment']['horizontal']['#description'] = t('Places items starting in the upper left and moving right.');
+    $form['alignment']['vertical']['#description'] = t('Places items starting in the upper left and moving down.');
+
+    $form['deprecated_table'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Use table (deprecated)'),
+      '#description' => t('Some sites might have made style changes based on the deprecated grid based on table. Only select this if absolutely needed.'),
```

Comment:
I'm also not sure about this description text, it seems quite verbose....  How about:
`For sites with views grids that need to use the deprecated HTML tables.`

Reply:
Fixed